### DISCUSSION TITLE
Added compatibility for latest Mac

### DISCUSSION THREAD

**Discussion by outofstyle47** [BepInEx/BepInEx#895] [BepInEx 6 proposals]
Hey! I noticed an announcement saying that there would be no more support for x86 Mac, and this is not about that. Tried getting this to work for the RavenM mod and got this error log after executing that said, "(have 'x86_64', need 'arm64e' or 'arm64')". Is this something you guys can add? I saw something about Doorstep 4 when searching for similar discussions but I think these are two different prospects.

**Comment by Dastano** [comment]
+1 for this, need some love

**Comment by toebeann** [comment]
Doorstop is what injects BepInEx into the game. If doorstop doesn't support arm64, BepInEx doesn't.

**Reply by JoeBalker** [reply]
This is actually quite an easy fix, I had this issue too. What you do is you right click Terminal.app in finder, click get info and turn on 'Open using Rosetta', and do the same to Ravenfield.app, then run the terminal commands again after restarting terminal.
